data/data/powervs/loadbalancer/providers.tf

```diff
@@ -1,8 +1,18 @@
+# TODO(mjturek): Remove this file once no longer needed
+
 terraform {
   required_providers {
     ibm = {
       source  = "ibm-cloud/ibm"
-      version = "~> 1.24.0"
+      version = "1.25.0"
+    }
+    ignition = {
+      source = "community-terraform-providers/ignition"
+      version = "2.1.2"
+    }
+    presign = {
+      source = "ibm/openshift-powervs/presign"
+      version = "0.0.1"
```

Comment:
do we really need this here in the iaas dir?

Reply:
See above, removed